site_scons/site_tools/doneapi.py

```diff
@@ -18,22 +18,19 @@ class DetectCompiler():
 
     def __init__(self):
         root = '/opt/intel/oneapi/compiler/latest'
-        binp = os.path.join(root, 'linux', 'bin')
-        libp = os.path.join(root, 'linux', 'lib')
-        include = os.path.join(root, 'linux', 'include')
-        binarch = os.path.join(binp, 'intel64')
-        libarch = os.path.join(root, 'linux', 'compiler', 'lib', 'intel64_lin')
+        binp = os.path.join(root, 'bin')
+        libp = os.path.join(root, 'lib')
+        include = os.path.join(root, 'include')
```

Comment:
I just want to point out that this is not backwards compatible with older versions, right?

Reply:
That's correct.  If we have a use-case for using an older version, I suppose some backward compatibility could be added.

Reply:
I think the question to ask here is why do we have icc support in scons and the answer is not so we can build with icc in CI, the answer is that we want users to be able to use icc with daos and therefore it should be obvious that backwards compatibility is desirable here.  Like Brian says something to address in master however.

Reply:
But surely everyone would want to be using the most recent icc wouldn't they?

Reply:
want to certainly, but as we've demonstrated they might not be and may want to be updating daos without forcing a system software change.